test/promql/tests/recording/remotesecret_availability_recording_rules_test.yml

```diff
@@ -0,0 +1,17 @@
+evaluation_interval: 1m
+
+rule_files:
+- remotesecret_availability_recording_rules.yaml
+
+tests:
+- interval: 1m
+  name: RSExporterTest
+  input_series:
+    - series: "redhat_appstudio_remotesecret_secretstorage_system_available{service='remote-secret-controller-manager-metrics-service'}"
+      values: "1 1 1 1 1"
+  promql_expr_test:
+    - expr: konflux_up
+      eval_time: 5m
+      exp_samples:
+        - labels: konflux_up{service='remote-secret-controller-manager-metrics-service', check='secretstorage'}
+          value: 1
```

Comment:
Need also a test case for when 0 is returned. Can `redhat_appstudio_remotesecret_secretstorage_system_available` return 0?